<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE launch>
<launch>
  <rosparam command="load" file="$(find mvp_grasping)/cfg/mvp_grasp.yaml" />
  <node name="grasp_entropy_node" pkg="mvp_grasping" type="grasp_entropy_node.py" output="screen"/>
</launch>
